content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -150,7 +150,7 @@ RuntimeClass Beta (v1.14) included the following changes:
   meaning it can no longer container `.` characters (in all versions). Valid handlers match the
   following regular expression: `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`.
 
-If you were using RuntimeClass Alpha (prior to v1.14), the following **actions are required**:
+**Action Required:** If you were using RuntimeClass Alpha (prior to v1.14), the following actions are required upon upgrade to v1.14:
```

Comment:
I re-read the page again and found a few nits as well as suggested changes for readability. Totally optional.
nit: The following actions are required to upgrade from the alpha version of the RuntimeClass feature to the beta version: